components/Meme.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect } from "react";
+
+
+function Meme() {
+    const [meme, setMeme] = React.useState({
+        topText: "",
+        bottomText: "",
+        randomImage: "http://i.imgflip.com/1bij.jpg" 
+    })
+    const [allMemeImages, setAllMemeImages] = React.useState([])
```

Comment:
I suggest to import `useState` as you did for the `useEffect` and then have your code improved like this: 
```suggestion
    const [meme, setMeme] = useState({
        topText: "",
        bottomText: "",
        randomImage: "http://i.imgflip.com/1bij.jpg" 
    })
    const [allMemeImages, setAllMemeImages] = useState([])
```